csharp/OnnxRuntime.CSharp.proj

```diff
@@ -72,7 +72,7 @@ CMake creates a target to this project
     </Exec>
   </Target>
 
-   <Target Name="ObtainPackageVersion" BeforeTargets="CreatePackage">
+   <Target Name="ObtainPackageVersion" BeforeTargets="CreatePackage;CreateWindowsAIPackage">
```

Comment:
Odd that the cross plat CSHARP project is what builds the Windows AI package?

Reply:
CSharp is a lie... it actually builds the native nuget packages.